Sming/Libraries/SDCard/SDCard.cpp

```diff
@@ -83,7 +93,7 @@ void SDCard_begin(uint8 PIN_CARD_SS)
 /*-------------------------------------------------------------------------*/
 /* Platform dependent macros and functions needed to be modified           */
 /*-------------------------------------------------------------------------*/
-static inline void dly_us (UINT n)	/* Delay n microseconds (avr-gcc -Os) */
+static inline void dly_us(UINT n) /* Delay n microseconds (avr-gcc -Os) */
```

Comment:
private functions can also be placed inside nameless namespace as alternative to `static`